### ISSUE TITLE
Fix GH Pages doc deployment

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#359]
Looks like the latest master is not able to find the sphinx package during the deployment stage on Travis.  As it is written, this stage happens only for master, after all the testing that is done in develop. I believe this is the first time this stage has been run since May, and with all the changes I'm not surprised it didn't work the first time.

Documentation building is actually tested during development, but that is within the tox environment which we've controlled, not outside (ie on Travis directly which is how the `deploy` is coded). I think this happened because I moved us away from using Conda in travis, since conda was already covered in azure tests. I think I just need to add a sphinx install to travis.

**Comment by janden** [comment]
So Conda has sphinx installed by default? That's strange.

Does this mean we need another version bump?

**Comment by garrettwrong** [comment]
The original code from year ago setup and installed a single conda _aspire_ env directly on the Travis machine. I reduce the environment down to a minimal `pip install tox`, but apparently missed thinking about sphinx in the base platform (my mistake).  TBH, our implementation of this doc deployment is a little strange to me (we reinvented wheel a little), but its probably easier to just patch it up with a line at this point.

This only effects the deployed docs, which haven't been up to date since I worked here, so rushing it in wasn't a huge concern.  I will be submitting a version bump for develop anyway, but I can do it in this PR if we want, np.
